pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -156,6 +156,10 @@
         @Parameter(names = { "-b",
                 "--batch-time-window" }, description = "Batch messages in 'x' ms window (Default: 1ms)")
         public double batchTimeMillis = 1.0;
+        
+        @Parameter(names = { "-bn",
+                "--batch-max-msgs" }, description = "Number of max messages in batch. (Default: 0 (disable))")
+        public int batchMaxMsgs = 0;
```

Comment:
Leaving the default to 0 is a bit confusing since it will not "disabling" the batching, nor the default max batch size of 1000 messages. Should we instead use 1000 as default here?

Reply:
ok..added default values.